### PR TITLE
[v2.5.8] Release feature charts

### PR DESCRIPTION
**Pull request by PennyScissors**

### CODE REVIEW

**Comment by cbron** [comment]
@PennyScissors can you remake this with changed in dev-v2.5

File: index.yaml [hunk 1]
```diff
@@ -1158,12 +1503,12 @@ entries:
       catalog.rancher.io/namespace: cattle-istio-system
       catalog.rancher.io/release-name: rancher-kiali-server
     apiVersion: v2
-    appVersion: v1.29.0
-    created: "2021-03-03T21:31:14.807153-07:00"
```
Comment (cbron) [inline index.yaml:1162]: These deletions are odd, but it looks like its just a weird way git did re-ordering. The original digest is still there. Can you confirm.

Comment (superseb) [reply index.yaml:1162]: v1.29.0 (https://github.com/rancher/charts/pull/1164/files#diff-08c65c1b5ef8384ef0200558777c52f4110afcbbacc880ac5375d555ca97ee3eR1541) and v.1.24.0 (https://github.com/rancher/charts/pull/1164/files#diff-08c65c1b5ef8384ef0200558777c52f4110afcbbacc880ac5375d555ca97ee3eR1576) are below

**Review by cbron — CHANGES_REQUESTED** [review]
Needs 4 more images that aren't tagged as RCs. No change needed yet in this PR. Need to wait.

**Comment by cbron** [comment]
Closing, will be handled with diff PR

**Pull request closed without merging**